pages/installation.md

```diff
@@ -176,30 +176,15 @@ of whether you are using mason or others, you can use this
 configuration below as a reference:
 
 ```lua
-    local lspconfig = require("lspconfig")
-    local configs = require("lspconfig.configs")
-
-    local lexical_config = {
-      filetypes = { "elixir", "eelixir", "heex" },
-      cmd = { "/my/home/projects/_build/dev/package/lexical/bin/start_lexical.sh" },
-      settings = {},
-    }
-
-    if not configs.lexical then
-      configs.lexical = {
-        default_config = {
-          filetypes = lexical_config.filetypes,
-          cmd = lexical_config.cmd,
-          root_dir = function(fname)
-            return lspconfig.util.root_pattern("mix.exs", ".git")(fname) or vim.loop.os_homedir()
-          end,
-          -- optional settings
-          settings = lexical_config.settings,
-        },
-      }
-    end
-
-    lspconfig.lexical.setup({})
+require('lspconfig').lexical.setup {
+  cmd = { "my/home/projects/_build/dev/package/lexical/bin/start_lexical.sh" },
+  root_dir = function(fname)
+    return util.root_pattern("mix.exs", ".git")(fname) or vim.loop.cwd()
+  end,
```

Comment:
```suggestion
```
As I understand it, these lines are not necessary because the default config already contains similar settings, https://github.com/neovim/nvim-lspconfig/blob/97e72e3a3bd3ae2034c2884da1253341409c2076/lua/lspconfig/server_configurations/lexical.lua#L6-L8

Reply:
i am not sure; i think config in Lexical's own docs needs to be integral, and it kind of replaces / is alternative to what you are linking from `github.com/neovim/nvim-lspconfig`. so personally, i would keep this part in the PR. if you still want to remove it, please confirm @scottming.

Reply:
It is fine